core/templates/dev/head/admin/Admin.js

```diff
@@ -30,6 +30,25 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
 
   $scope.currentTab = $scope.TAB_ACTIVITIES;
 
+  $scope.$watch(function() {
+    return window.location.hash;
+  }, function(newHash) {
+    switch (newHash) {
+      case '#jobs':
```

Comment:
Can we pull out these URLs as constants, and use them in the HTML code too? This would help prevent people from changing the html and breaking the tab reloading.